Content.Shared/Silicons/StationAi/StationAiCoreComponent.cs

```diff
@@ -39,6 +40,19 @@ public sealed partial class StationAiCoreComponent : Component
     public EntProtoId? PhysicalEntityProto = "StationAiHoloLocal";
 
     public const string Container = "station_ai_mind_slot";
+    
+    // Starlight-start: Linking with ai upload console
+    
+    [DataField, AutoNetworkedField]
+    public NetEntity? LawConsole;
```

Comment:
```suggestion
    public EntityUid? LawConsole;
```

NetEntity can't be DataField